source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -65,11 +65,13 @@ Seedlist Discovery
 ------------------
 
 In this preprocessing step, the driver will query the DNS server for SRV
-records on ``{hostname}.{domainname}``, prefixed with ``_mongodb._tcp.``:
-``_mongodb._tcp.{hostname}.{domainname}``. This DNS query is expected to
-respond with one or more SRV records. From the DNS result, the driver now MUST
-behave the same as if an ``mongodb://`` URI was provided with all the host
-names and port numbers that were returned as part of the DNS SRV query result.
+records on ``{hostname}.{domainname}``, prefixed with the SRV service name
+and protocol. The SRV service name is provided in the ``srvServiceName`` URI option and
+defaults to ``mongodb``. The protocol is always ``tcp``. After prefixing, the URI
+should look like: ``_{srvServiceName}._tcp.{hostname}.{domainname}``. This DNS query
+is expected to respond with one or more SRV records. From the DNS result, the driver
+now MUST behave the same as if an ``mongodb://`` URI was provided with all the host names
+and port numbers that were returned as part of the DNS SRV query result.
```

Comment:
This is actually technically not true as the default options for `mongodb://` and `mongodb+srv://` are different. For example `mongodb+srv://` implies `&tls=true` whereas `mongodb://` does not. As well `mongodb+srv://` must read additional options from the associated TXT record whereas `mongodb://` does not. So this statement can be misinterpreted. Please clarify.

Reply:
That's fair. I didn't write this language, but I've changed it to be more specific and not claim that drivers must "behave the same as a `mongodb://` URI was provided".

Reply:
This occurred to me while I was working on https://github.com/mongodb/specifications/pull/1069 as well, since I ended up modifying this paragraph in that PR. I think we should just state that the results get added to the seedlist and leave it at that.